x-pack/plugins/observability/public/pages/alerts/components/alerts_flyout/types.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { AlertsTableFlyoutBaseProps } from '@kbn/triggers-actions-ui-plugin/public';
+import { ParsedTechnicalFields } from '@kbn/rule-registry-plugin/common/parse_technical_fields';
+import { ParsedExperimentalFields } from '@kbn/rule-registry-plugin/common/parse_experimental_fields';
+import { EcsFieldsResponse } from '@kbn/rule-registry-plugin/common/search_strategy';
+
+export type FlyoutProps = AlertsTableFlyoutBaseProps & {
```

Comment:
nit: there's no need to put this type in another file. the types are easier to read when they are next to the code.